test/Microsoft.Sbom.Api.Tests/Workflows/ManifestGenerationWorkflowTests.cs

```diff
@@ -402,9 +402,13 @@ await externalDocumentReferenceChannel.Writer.WriteAsync(new ExternalDocumentRef
 
             var organizationElements = elements.Where(element => element["type"].ToString() == "Organization").ToList();
             Assert.AreEqual(2, organizationElements.Count);
+
```

Comment:
(nit): Ordering here feels off. I'd expect us to consistently follow one of these patterns
1. All the vars get set, then all of the Asserts
2. Set 1 var, then Assert, (and probably leave an empty newline)

Asserts 1 and 4 follow pattern 1, while Asserts 2 and 3 follow pattern 2